src/data/waypoint.json

```diff
@@ -66,17 +66,20 @@
 			"name": "General Documentation",
 			"path": "docs",
 			"shortName": "Documentation",
+			"addOverviewItem": true,
```

Comment:
Same as in `vault.json`, making sure to retain existing behaviour with `addOverviewItem` on all `rootDocsPaths`